portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments=new ArrayList<String>();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment=request.getParameter("comment");
```

Comment:
Format: final String comment = request.getParameter("comment");

FYI - You may want to use some auto format tool, such as https://github.com/google/google-java-format